src/models/apartment/index.js

```diff
@@ -55,6 +50,13 @@ const Apartment = sequelize.define('Apartment', {
   },
 }, {
   paranoid: true,
+  hooks: {
+    beforeValidate: (apartment) => {
+      if (apartment.CityId && !apartment.addressCity) {
+        apartment.addressCity = _.capitalize(apartment.CityId);
```

Comment:
Est-ce qu'on ne mettrait pas cette règle pour tous les `addressCity` ? 
```suggestion
        apartment.addressCity = apartment.CityId;
```